flow.py

```diff
@@ -142,6 +144,8 @@ def create_packet(self, packet_id, delay=0.0):
         event1_message = "flow.create_packet: Flow" + str(self.flow_id) + \
             ": made data packet " + str(new_packet.packet_id)
         self.ns.add_event(event1, event1_message, delay=delay)
+        self.ns.record_packet_send_time(self.flow_id, new_packet.packet_id)
```

Comment:
I guess since the host is the one that sends the packet we'd need the host to be the one to log for the flow when the packet get sent. idk if doing that is better than just loging it here.